docs/azure/sdk/logging.md

```diff
@@ -154,44 +159,29 @@ Since the `Logging:LogLevel:Azure.Messaging.ServiceBus` key is set to `Debug`, S
 
 ### Logging without client registration
 
-While this approach is far less common, your app may not need to explicitly register clients for Azure SDK libraries. In that scenario, a call to `AddAzureClientsCore` will suffice. Take this approach when your app uses ASP.NET extension libraries that depend on other Azure SDK libraries. Common examples of such ASP.NET extension libraries include:
-
-- [Azure Key Vault key encryptor for DataProtection](/dotnet/api/overview/azure/Extensions.AspNetCore.DataProtection.Keys-readme)
-- [Azure Key Vault secrets configuration provider](/dotnet/api/overview/azure/Extensions.AspNetCore.Configuration.Secrets-readme)
-- [Azure Blob Storage key store for DataProtection](/dotnet/api/overview/azure/Extensions.AspNetCore.DataProtection.Blobs-readme)
+There are scenarios in which [registering an Azure SDK library's client with the dependency injection (DI) container](dependency-injection.md#register-client) is either impossible or unnecessary:
 
-Consider the following *Program.cs* example, which uses the two aforementioned [ASP.NET Core Data Protection](/aspnet/core/security/data-protection/introduction) extension libraries:
+- The Azure SDK library doesn't include an `IServiceCollection` extension method to register a client in the DI container.
+- Your app uses ASP.NET extension libraries that depend on other Azure SDK libraries. Examples of such ASP.NET extension libraries include:
+  - [Azure Key Vault key encryptor for DataProtection](/dotnet/api/overview/azure/Extensions.AspNetCore.DataProtection.Keys-readme)
+  - [Azure Key Vault secrets configuration provider](/dotnet/api/overview/azure/Extensions.AspNetCore.Configuration.Secrets-readme)
+  - [Azure Blob Storage key store for DataProtection](/dotnet/api/overview/azure/Extensions.AspNetCore.DataProtection.Blobs-readme)
 
-```csharp
-using Azure.Identity;
-using Microsoft.AspNetCore.DataProtection;
-using Microsoft.Extensions.Azure;
+In these scenarios, complete the following steps:
 
-// code omitted for brevity
+1. Manually start the log forwarder:
+    1. Invoke the parameterless `AddAzureClientsCore` method to register the log forwarder service with the DI container.
+    1. Fetch the log forwarder service singleton from the DI container and invoke its <xref:Microsoft.Extensions.Azure.AzureEventSourceLogForwarder.Start%2A> method.
```

Comment:
It would be good to show a snippet for how to get the log forwarder.
It should also be possible to inject the Log forwarder via the constructor of some other type that is being initialized through dependency injection, and to call Start on it from there, e.g. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusClientFactory.cs#L35